compiler/rustc_session/src/options.rs

```diff
@@ -2000,6 +2000,10 @@ options! {
         "enable queries of the dependency graph for regression testing (default: no)"),
     randomize_layout: bool = (false, parse_bool, [TRACKED],
         "randomize the layout of types (default: no)"),
+    regparm: Option<u32> = (None, parse_opt_number, [TRACKED],
+        "On x86-32 targets, setting this to N causes the compiler to pass N arguments \
+        in registers EAX, EDX, and ECX instead of on the stack.\
```

Comment:
This should follow whatever you take for the other:
```suggestion
        in registers EAX, EDX, and ECX instead of on the stack for\
        "C", "cdecl", and "stdcall" fn.\
```
Or:
```suggestion
        in registers EAX, EDX, and ECX instead of on the stack for some non-Rust ABIs.\
```